packages/components/src/button/tests/chromatic/ButtonGroup.chroma.jsx

```diff
@@ -0,0 +1,114 @@
+import { AddMajorIcon } from "@components/icons";
```

Comment:
This file should not be added. Its been renamed to .stories.tsx and this file added is the result of a merge conflict gone wrong